ui/js/services/perf/errors.js

```diff
@@ -0,0 +1,14 @@
+import treeherder from '../../treeherder';
+
+treeherder.factory('alertHttpError',
```

Comment:
If this move of `alertHttpError()` is being kept in this PR, I would prefer it to be a standard function that's just imported in graphs.js. It doesn't need to be an AngularJS factory.